uniMASK/sweeps/medium_maze_sweep_10.yaml

```diff
@@ -1,89 +0,0 @@
-program: train.py
```

Comment:
@orrp  I've removed all `.sh` files and sweep `.yaml` files relating to Maze that are not necessary to reproduce the paper. Have you done that too for minigrid? If not, probably worth doing a tiny cleanup of that kind there

Reply:
done https://github.com/micahcarroll/uniMASK/pull/3/commits/a9fd1fd70b5d25d55b95fb482f053f1491c286dd